.gitignore

```diff
@@ -24,7 +24,8 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
-*.mypy_cache/
+.mypy_cache/
```

Comment:
is removing the asterisk here intentional?

Reply:
Partly — I copied and pasted it along with line below. The `*` isn't doing anything; it's a dotfile rather than an extension